docs/sphinx/developers/generating-test-images.txt

```diff
@@ -91,6 +91,10 @@ To generate a fake file containing regions of interest:
     touch "regions&ellipses=20.fake"
     touch "regions&rectangles=5&lines=25.fake"
 
+Replace ``regions`` in the above examples with the desired image or plate which will contain the regions, e.g.
```

Comment:
Yes, but this PR will now conflict with https://github.com/openmicroscopy/bioformats/pull/2425/commits/f7319d4a380c201eb61913207c118511fef1c4c5